lib/puppet/provider/package/npm.rb

```diff
@@ -70,10 +70,24 @@ def install
       package = "#{resource[:name]}@#{resource[:ensure]}"
     end
 
+    extraopts = []
+
+    if resource[:install_options]
+      extraopts += resource[:install_options][0].collect do |k,v|
+        if k == 'registry'
+          extraopts += [ "--registry" , v ]
```

Comment:
Is this how the `install_options` for the package provider works? What about cases other than `registry`? Wouldn't it be better to just have keys like `{'--registry' => 'custom'}`?